src/lexedata/importer/cognates.py

```diff
@@ -63,14 +63,9 @@ def properties_from_row(
         nargs="?",
         type=Path,
         default="Wordlist-metadata.json",
-        help="Path to the metadata.json file (default: ./Wordlist-metadata.json)",
-    )
-    parser.add_argument(
-        "--debug-level",
```

Comment:
This switch did not have functionality yet, so it makes sense to take it out.
But actually, we should think about how to treat logging levels in all our scripts that use loggers.
What do you suggest? Could you have a look around how other programs do it, or whether there are existing solutions?
In the meeting agenda, I mentioned [typer](https://github.com/tiangolo/typer), which has the explicit goal of making CLI programs better, maybe those guys or their buddies have an opinion on logging, too?